web/src/main/java/org/akaza/openclinica/control/managestudy/RemoveStudyEventServlet.java

```diff
@@ -97,7 +97,8 @@ public void processRequest() throws Exception {
 
             StudyDAO studydao = new StudyDAO(sm.getDataSource());
             StudyBean study = (StudyBean) studydao.findByPK(studySub.getStudyId());
-            request.setAttribute("study", study);
+            // @sinzii: changed attr name from study to subStudy
```

Comment:
Please remove this comment.  It is not explaining functionality and is not needed.